.editorconfig

```diff
@@ -1,30 +0,0 @@
-# http://editorconfig.org
```

Comment:
I think we can keep it ? It the same file as MJML repo and written components will be compliant with MJML code style.